bin/build_utils

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -euo pipefail
+
+####
+# Functions to generate version numbers for this project
+####
+
+readonly VERSION_GO_FILE="pkg/trace/version.go"
+
+function git_version() {
+  grep -v '^//' "${VERSION_GO_FILE}" | grep 'const Version =' | awk -F'= ' '{print $2}' | tr -d '"'
```

Comment:
If the `version.go` is preserved, the following suggestion will simplify extracting the version.
```suggestion
  grep 'const Version' "${VERSION_GO_FILE}" | awk -F'=' '{print $2}' | xargs
```

Here's an example of how xargs interprets the strings for you, and how that simplifies the rest of the command as well:
```shell
$ echo -n "   \t'foo' \n\n '\t'" | xargs
foo
```